### PR TITLE
TMP-261: Update design of digital contract component

### PR DESCRIPTION
**Pull request by robinsalehjan**
# Why?

- We want to do a small redesign of the digital contract component. Instead of creating a custom component I opted to build it into the `ContractActionView` class. 

# What?

- Add optional `title` and `subtitle` to `ContractActionView` class
- Add `trailingImage` and `paragraphSpacing` parameters to `configure` method
  - If `trailingImage` is present, the image will be added to the top right corner of the view.
- Add missing assets to Demo and Project.

# Version Change

Minor

# UI Changes

| iPhone 6s | iPhone 12 PM |
| --- | --- |
| iPhone 6s | iPhone 12 PM |

### CODE REVIEW

File: FinniversKit/Sources/Components/ContractActionView/ContractActionView.swift [hunk 1]
```diff
@@ -82,3 +133,9 @@ public class ContractActionView: UIView {
         delegate?.contractActionView(self, didSelectActionButtonWithUrl: buttonUrl)
     }
 }
+
+private extension UIColor {
+    class var backgroundCircleColor: UIColor {
+        UIColor(red: 0.89, green: 0.89, blue: 0.91, alpha: 1.00)
```
Comment (ag12) [inline FinniversKit/Sources/Components/ContractActionView/ContractActionView.swift:139]: Finnes det ikke egen fargekode for denne?

**Review by ag12 — APPROVED** [review]
gj.

File: FinniversKit/Sources/Components/ContractActionView/ContractActionView.swift (hunk 1 above)
Comment (robinsalehjan) [reply FinniversKit/Sources/Components/ContractActionView/ContractActionView.swift]: Denne skulle ikke med, noen rester igjen fra testing. Har fjernet den nå.

**Comment by ag12** [comment]
herlig.

**Pull request merged**